RNTester/RNTesterUnitTests/RCTBundleURLProviderTests.m

```diff
@@ -20,12 +20,24 @@
 
 static NSURL *localhostBundleURL()
 {
-  return [NSURL URLWithString:[NSString stringWithFormat:@"http://localhost:8081/%@.bundle?platform=%@&dev=true&minify=false&app=com.apple.dt.xctest.tool", testFile, kRCTPlatformName]]; // TODO(macOS ISS#3536887)
+  return [NSURL
+      URLWithString:
+          [NSString
+              stringWithFormat:
+                  @"http://localhost:8081/%@.bundle?platform=%@&dev=true&minify=false&modulesOnly=false&runMdoule=true&app=com.apple.dt.xctest.tool",
+                  testFile,
+                  kRCTPlatformName]]; // TODO(macOS GH#774)
 }
 
 static NSURL *ipBundleURL()
 {
-  return [NSURL URLWithString:[NSString stringWithFormat:@"http://192.168.1.1:8081/%@.bundle?platform=%@&dev=true&minify=false&app=com.apple.dt.xctest.tool", testFile, kRCTPlatformName]]; // TODO(macOS ISS#3536887)
```

Comment:
Same here- the bundle URL is changing but I saw we had a `macOS` todo fix here so just want to verify that the macOS fix wasn't related to the local host URL that you're changing?

Reply:
The macOS TODO is related to the addition of `kRCTPlatformName`. Upstream, this is hard-coded as `ios`, but naturally this will be different on macOS.